globus_cli/commands/endpoint/server/list.py

```diff
@@ -14,6 +17,28 @@ def server_list(endpoint_id):
     """
     client = get_client()
 
+    endpoint = client.get_endpoint(endpoint_id)
+
+    if endpoint['host_endpoint_id']:  # not GCS -- this is a share endpoint
+        raise click.UsageError(
+            "{id} ({display_name}) is a share and does not have servers.\n"
+            "\n"
+            "To see details of the share, use\n"
+            "    globus endpoint show {id}\n"
+            "\n"
+            "To list the servers on the share's host endpoint, use\n"
+            "    globus endpoint server list {host_endpoint_id}".
+            format(**endpoint.data)
```

Comment:
As a stylistic note, I would have written this utterly differently (multiline string with `textwrap.dedent`, if it matters).
This way is fine, but should we set a convention on how long messages are represented/handled so that we have  a consistent style?